src/modules/project/pages/projectFunding/views/refundPayoutRsk/PayoutRsk.tsx

```
@@ -471,6 +474,17 @@ export const PayoutRsk: React.FC<PayoutRskProps> = ({
   })
   const isPrismPayout = contractType === PayoutContractType.Prism
   const requiresInternalLock = isPrismPayout || isInternalRetryPayout
+  const { data: projectRskEoaMetadataData } = useProjectRskEoaMetadataQuery({
+    variables: { where: { id: project.id } },
+    skip: !isOpen || !project.id || !rskAddress || !isPrismPayout,
+    fetchPolicy: 'cache-first',
+  })
+  const storedProjectRskEoa = projectRskEoaMetadataData?.projectGet?.rskEoas.find((projectWallet) => {
+    return rskAddress
+      ? projectWallet.rskAddress.toLowerCase() === rskAddress.toLowerCase()
+      : projectWallet.isCurrent
```

ESLint

[error] Replace `⏎······?·projectWallet.rskAddress.toLowerCase()·===·rskAddress.toLowerCase()⏎·····` with `·?·projectWallet.rskAddress.toLowerCase()·===·rskAddress.toLowerCase()`

(prettier/prettier)